config/nextflow_sh.config

```diff
@@ -47,6 +47,9 @@ profiles {
             withLabel:Fastqc {
                 module='fastqc/0.11.8'
             }
+            withLabel:Mosdepth {
+                module='mosdepth/0.3.8'
```

Comment:
```suggestion
```
I don't think Milton has a mosdepth module. Can you remove the mosdepth block for now.